root/edit/details/EditInstrument.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import IntentionallyRawIcon
+  from '../components/IntentionallyRawIcon';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import FullChangeDiff from
+  '../../static/scripts/edit/components/edit/FullChangeDiff';
+import WordDiff from '../../static/scripts/edit/components/edit/WordDiff';
+
+type EditInstrumentEditT = {
+  ...EditT,
+  +display_data: {
+    +comment?: CompT<string | null>,
+    +description?: CompT<string | null>,
+    +instrument: InstrumentT,
+    +name?: CompT<string>,
+    +type?: CompT<InstrumentTypeT | null>,
+  },
+};
+
+type Props = {
+  +edit: EditInstrumentEditT,
+};
+
+const EditInstrument = ({edit}: Props): React.Element<'table'> => {
+  const display = edit.display_data;
+  const name = display.name;
+  const comment = display.comment;
+  const type = display.type;
+  const description = display.description;
+  const rawIconSection = (
+    <>
+      {' '}
+      <IntentionallyRawIcon />
+    </>
+  );
+
+  return (
+    <table className="details edit-event">
+      <tr>
+        <th>{addColonText(l('Instrument'))}</th>
+        <td colSpan="2"><EntityLink entity={display.instrument} /></td>
+      </tr>
+      {name ? (
+        <WordDiff
+          extraNew={rawIconSection}
+          extraOld={rawIconSection}
+          label={addColonText(l('Name'))}
+          newText={name.new}
+          oldText={name.old}
+        />
+      ) : null}
+      {comment ? (
+        <WordDiff
+          label={addColonText(l('Disambiguation'))}
+          newText={comment.new ?? ''}
+          oldText={comment.old ?? ''}
+        />
+      ) : null}
+      {type ? (
+        <FullChangeDiff
+          label={addColonText(l('Type'))}
+          newContent={type.new
+            ? lp_attributes(type.new.name, 'instrument_type')
+            : null}
+          oldContent={type.old
+            ? lp_attributes(type.old.name, 'instrument_type')
+            : null}
+        />
+      ) : null}
+      {description ? (
+        <WordDiff
+          extraNew={rawIconSection}
+          extraOld={rawIconSection}
```

Comment:
Do not display the icon for empty values:

```suggestion
          extraNew={nonEmpty(newDescription) ? rawIconSection : null}
          extraOld={nonEmpty(oldDescription) ? rawIconSection : null}
```

Copied from https://github.com/metabrainz/musicbrainz-server/commit/65090b8089f4f6b65717a0fc54c465aa3aeb3b83#diff-8acf88c2a9626fe10451af2e3c563047c241ad9937c38293c36b0ccd70cdcf7eR77-R78

Reply:
Good catch!